testFramework/com/twitter/intellij/pants/testFramework/OSSPantsIntegrationTest.java

```diff
@@ -8,14 +8,21 @@
 import com.intellij.execution.configurations.RunConfiguration;
 import com.intellij.execution.impl.RunManagerImpl;
 import com.intellij.execution.impl.RunnerAndConfigurationSettingsImpl;
+import com.intellij.openapi.editor.Document;
```

Comment:
no new functions, only to move some helper functions up from `OSSFileSyncIntegrationTest.java`